includes/Performance.php

```diff
@@ -360,4 +362,17 @@ public function isJetPackBoostActive() {
 
 		return $exists;
 	}
+
+	/**
+	 * Set default values for JetPack Boost.
+	 *
+	 * @return void
+	 */
+	public function set_default_values_for_jetpack_boost() {
+		$isFreshInstall = container()->get( 'isFreshInstallation' );
```

Comment:
```suggestion
		$is_fresh_install = container()->get( 'isFreshInstallation' );
```
Snake case for variable names.

Reply:
ok, tks